lua/terrortown/entities/roles/jester/winstates.lua

```diff
@@ -179,7 +179,7 @@ function JesterWinstateFive(ply, killer)
 
 	for _, v in pairs(roles.GetList()) do
 		if v.defaultTeam == rd.defaultTeam or v.defaultTeam == TEAM_JESTER then
-			reviveRoles[v] = nil
+			reviveRoles.remove(v)
```

Comment:
`remove` needs the index. Maybe let it with `reviveRoles[v] = nil`, that's better with performance and makes more sense for not enumerated lists (i forgot that the return is a table, this should be changed theoretically) - if I think about it.